cloud_governance/policy/azure/cleanup/instance_idle.py

```diff
@@ -0,0 +1,65 @@
+
+from cloud_governance.common.utils.configs import INSTANCE_IDLE_DAYS
+from cloud_governance.common.utils.utils import Utils
+from cloud_governance.policy.helpers.azure.azure_policy_operations import AzurePolicyOperations
+
+
+class InstanceIdle(AzurePolicyOperations):
```

Comment:
Do you think we should have 1 abstract class for IdleInstance ?

Reply:
Yes, true. yet to add the abstract class for it.

Reply:
I think we should add it, after it will be more complex
Also I think we should change the class name InstanceIdle => IdleInstance

Reply:
I think, it should be better if we service name as prefix.

Reply:
What do you mean ?

Reply:
I mean Instance is a service idle is an activity. So InstanceIdle will justify the service activity.